docs-v1/smart-contracts/events.md

```diff
@@ -0,0 +1,242 @@
+# Events
+
+## IAllowListClaim
+```text
+// Issued when Vault or Pools set or clear the emergency paused state
```

Comment:
Change comment line to reflect accurate description. Left over comment from Balancer reference over looked